.circleci/test-deploy.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       - mapi/scan:
           api-url: "https://demo-api.mayhem4api.forallsecure.com/api/v3"
           api-spec: "https://demo-api.mayhem4api.forallsecure.com/api/v3/openapi.json"
+          target: "forallsecure/mapi-circleci-orb"
```

Comment:
@swarkentin How should this be specified now?  I am getting a 403 in my tests with `forallsecure/mapi-circleci-orb`

Reply:
To target a workspace, use the fully qualified path of:

`workspace/project/target`

So... how about:

`forallsecure/integrations/mapi-circleci-orb`

?

Reply:
thank you